src/main/java/pers/gwyog/gtneioreplugin/util/CSVMaker.java

```diff
@@ -24,9 +24,7 @@ public CSVMaker() {}
     public static List<Oremix> Combsort(List<Oremix> liste) {
         try {
             List<Oremix> liste2 = new ArrayList<Oremix>(liste.size());
```

Comment:
So, all of those variable names are in German 😄 Would you be willing to refactor this into English?

Reply:
Didn't even know it's German 🤣